interpreter/text/parser.mly

```diff
@@ -907,6 +912,11 @@ expr1 :  /* Sugar */
       let tag = $3 c tag in
       let hs, es = $4 c in
       es, resume_throw x tag hs }
+  | RESUME_THROW_REF var resume_expr_handler
```

Comment:
What is the difference between the two different additions in this file?

Reply:
@tlively, the former production handles unparenthesised instructions. For instructions with optional or list-like tail immediates such duplication is necessary due to the limitations of LALR(1) grammars.